pkg/iac/terraform/presets.go

```diff
@@ -2,12 +2,23 @@ package terraform
 
 import (
 	"fmt"
+	"math/rand/v2"
 	"strings"
 
 	"github.com/google/uuid"
 	"github.com/zclconf/go-cty/cty"
 )
 
+var resourceRandomAttributes = map[string][]string{
+	// If the user leaves the name blank, Terraform will automatically generate a unique name
```

Comment:
You left `random_pet` behind 😿

Reply:
`random_pet` has only an id attribute, which we fill in for all resources :)